lib/tests/streamlit/util_test.py

```diff
@@ -145,15 +145,13 @@ def test_unsuccessful_index_(self, input, find_value):
             ),
         ]
     )
-    def test_drop_key_query_params(
+    def test_drop_key_in_dict(
```

Comment:
nit: this is a weird test name given that it doesn't include the name of the function being tested